apiserver/worker/compiler.py

```diff
@@ -489,6 +493,14 @@ def compile(self, bot_dir, globs, errors, timelimit):
         [BOT],
         [([""], ExternalCompiler(comp_args["Haskell"][0]))]
     ),
+    Language("Haskell/Stack", BOT, "stack.yaml",
```

Comment:
Would a bot using still be likely to include a `MyBot.hs` file in the root directory of the zip? If it does you'll need to add a language disambiguation section in `dectect_language` like Snaar did for cmake based builds.

Reply:
Based on what @wizzup provided in https://github.com/HaliteChallenge/Halite-II/issues/265#issuecomment-346775263, it seems not.

Reply:
Ah, yep that should work fine then.